### ISSUE TITLE
Update eslint

### ISSUE THREAD

**Issue by simison** [meanjs/mean#1846]
I feel that the best way to start moving towards v1.0 (#1666) is to first update eslint to latest.

- [ ] update to eslint 3.x,
- [ ] get rid of `eslint-config-airbnb`, 
- [ ] set CI builds fail on linting errors

**Comment by lirantal** [comment]
isn't `eslint-config-airbnb` used as part of applying the rules through eslint?

**Comment by simison** [comment]
@lirantal True, but it's geared towards React native and [it's deprecated](https://www.npmjs.com/package/eslint-config-airbnb#eslint-config-airbnblegacy), too.

**Comment by simison** [comment]
Actually if we want a predefined set of eslint rules, we should consider [XO](https://www.npmjs.com/package/xo).

**Comment by lirantal** [comment]
XO is very opinionated by a specific set of people. If we're at a cross-road to diverge how about [standardjs](https://standardjs.com) ?
